x/auth/client/cli/tx_multisign.go

```diff
@@ -303,8 +341,42 @@ func makeBatchMultisignCmd() func(cmd *cobra.Command, args []string) error {
 				PubKey:        pubKey,
 			}
 
+			anyPk, err := codectypes.NewAnyWithValue(multisigPub)
```

Comment:
cant comment above but should this command move to /tx and auth depend on /tx to get the command or that is not needed?

Reply:
I'm not seeing any obvious wins from moving the command to x/tx did you have something in mind?

Reply:
yea for sure, makes to do in a follow up

Reply:
https://github.com/cosmos/cosmos-sdk/issues/15888